styles.css

```diff
@@ -0,0 +1,1240 @@
+
+}
+
+.side {
+    grid-area: side;
+    background-color: white;
+    ;
+}
+
+.left-header {
+    grid-area: left-header;
+    margin-right: 70%;
+    margin-left: 10%;
+    margin-top: 5%;
+
+}
+
+.left-header img {
+    object-fit: contain;
+    height: 100%;
+    width: 100%;
+    border-radius: 60%;
+}
+
+.list-logo {
+    grid-area: list-logo;
+    display: grid;
+    grid-template-columns: repeat(2, auto);
+    margin-top: 4mm;
+    margin-left: 4mm;
+
+}
+
+.list-logo ul {
+    list-style: none;
+    line-height: 60px;
+}
+
+.list-logo img {
+    width: 25px;
+
+}
+
+.list-logo span {
+    font-weight: 550;
+    font-size: large;
+    font-family: Arial, Helvetica, sans-serif;
+
+}
+
+#home {
+    width: 115px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+}
+
+#explore {
+
+    width: 130px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+
+}
+
+#notifications {
+
+    width: 180px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#messages {
+
+    width: 155px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#bookmarks {
+
+    width: 165px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#lists {
+
+    width: 110px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#profile {
+
+    width: 125px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#more11 {
+
+    width: 110px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+
+#home:hover {
+    background-color: rgb(231 231 232);
+}
+
+#explore:hover {
+    background-color: rgb(231 231 232);
+}
+
+#notifications:hover {
+    background-color: rgb(231 231 232);
+}
+
+#messages:hover {
+    background-color: rgb(231 231 232);
+}
+
+#bookmarks:hover {
+    background-color: rgb(231 231 232);
+}
+
+#lists:hover {
+    background-color: rgb(231 231 232);
+}
+
+#profile:hover {
+    background-color: rgb(231 231 232);
+}
+
+#more11:hover {
+    background-color: rgb(231 231 232);
+}
+
+
+
+
+
+
+
+.tweak {
+    grid-area: tweak;
+    width: 88%;
+    color: white;
+    margin-left: 4%;
+    margin-right: 4%;
+    margin-bottom: 8mm;
+    border-radius: 30px;
+    background-color: rgb(29 156 240);
+    text-align: center;
+    padding-top: 5%;
+    font-size: 18px;
+    font-weight: bolder;
+    font-family: Arial, Helvetica, sans-serif;
+
+}
+
+.tweak:hover {
+    background-color: rgb(22 124 192);
+}
+
+.bottom-left {
+    grid-area: bottom-left;
+    margin-bottom: 5mm;
+    padding-top: 2mm;
+    background-color: white;
+    display: grid;
+    grid-template: "pic  info     dots"1fr/ 20% 60% 20%;
```

Comment:
This might be an issue with how github is displaying the code, but it looks like you might have a tab in your `grid-template` in-between info and dots. This shouldn't effect how your grid works, just make's it a little less readable